lib/index.d.ts

```diff
@@ -452,6 +453,21 @@ declare class Realm {
      * @returns void
      */
     write(callback: () => void): void;
+
+    /**
+     * @returns void
```

Comment:
If this applies to other methods than just this one, then yes. I didn't check for that. :-)